visual-e2e/examples/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test.visual.sauce": "npx wdio test/configs/wdio.visual.all.conf.js",
-    "test.visual.crossplatform": "npx wdio test/configs/wdio.visual.conf.js"
+    "test.visual.sauce": "npx wdio test/configs/wdio.shared.conf.js",
```

Comment:
Can you provide EU and US examples here and also add that in the readme? That's the structure how we do it for all examples in this repo

Reply:
Visual e2e only works in US